renderer/src/main.tsx

```diff
@@ -15,8 +16,19 @@ ReactDOM.createRoot(
   </React.StrictMode>
 )
 
-window.electron.onActivityLogged(entry => {
-  console.log('[ACTIVITY] %j', entry)
+const activities: Activity[] = []
+
+window.electron.onActivityLogged(activity => {
+  activities.push(activity)
+  activities.sort((a, b) => {
+    return a.timestamp !== b.timestamp
+      ? b.timestamp - a.timestamp
+      : a.id.localeCompare(b.id)
+  })
```

Comment:
if we moved sorting to the backend, then we wouldn't have to sort on every new event. Otherwise I think we could do it in either location, and having it on the view layer might be easier to get right wrt race conditions